src/Sources/Coalescence.jl

```diff
@@ -30,9 +30,9 @@ Updates the collision-coalescence integrals.
 """
 function update_coal_ints!(
```

Comment:
it looks like `update_coal_ints!` in [line 289](https://github.com/CliMA/Cloudy.jl/blob/a2e21a6347fd0fc242fa9192738ed5b788180c9a/src/Sources/Coalescence.jl#L289) has to be updated for the new type too :)